lib/ansible/modules/network/aci/aci_tenant_ep_retention_policy.py

```diff
@@ -66,7 +66,7 @@
     type: int
   description:
     description:
-    - Description for the End point rentention policy.
```

Comment:
I think we should keep this as is since rest of the documentation mentions like 'End Point'

Reply:
agree, done, thank you!